docs/developer/local_up_kubeedge.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Kubeedge local up script guide
+sidebar_position: 10
+---
+
+This page explains how to use shell script **hack/local-up-kubeedge.sh** in **kubeedge/kubeedge** repo to create a local KubeEdge cluster, installing necessary components, and starting CloudCore and EdgeCore services.
+
+## Usage Steps
+
+1. Preparation:
+Ensure that you have kind, kubectl, docker, and other necessary tools installed.
+Ensure that your system has a Go language development environment installed, and the version meets the requirements of KubeEdge.
+Based on the user's choice of container runtime (docker, cri-o, isulad), user can verify the prerequisites via the script below, or it will also be verified before the installation start in **hack/local_up_kubeedge.sh**
+
+```shell
+verify_docker_installed
+verify_crio_installed
+verify_isulad_installed
+kubeedge::golang::verify_golang_version
+check_kubectl
+check_kind
+```
+
+2. Executing the Script:
+Before running the script, you can modify some environment variables to suit your needs, such as **LOG_DIR** and **CONTAINER_RUNTIME** in the script.
+The script should be run with administrative privileges, as some operations (e.g., software installation, system configuration changes) require sudo access.
+
+```shell
+hack/local_up_kubeedge.sh
+```
+This script will run the environment setting, prerequesites verification, installation and so on. For details, please refer [Script Analysis](#script-analysis)
+
+3. Checking the Running Status:
+After the script completes, you can use kubectl commands to check the status of the cluster's nodes and the running status of KubeEdge components.
+
+```shell
+healthcheck
+kubectl get nodes | grep edge-node | grep -q -w Ready && break
+```
```

Comment:
In my opinion, healthcheck is generally only used inside scripts. Users can directly check the kubeedge node status through the k8s API, so I recommend deleting healthcheck here.

Reply:
Removed that part in the usage steps, and mentioned in the script analysis that user could use k8s API to check the health status.